packages/runtime/src/next/api-handler/index.ts

```diff
@@ -195,6 +198,13 @@ export function MakeswiftApiHandler(
         .exhaustive()
     }
 
+    if (matches('/webhook')) {
```

Comment:
Do we want to deprecate or remove the `/revalidate` endpoint now? 🤔

Reply:
I think so. This is all internal, though, so it would mostly be for us to not continue to extend it

Reply:
Sounds good. Added a new commit for this.

Reply:
I changed my mind. Let's keep this endpoint for now until we conditionally disable revalidation on the backend. I don't want to remove this endpoint if the backend is still sending revalidation requests.

Reply:
Sorry, I didn't clarify that my response was "I think so" to "deprecate", not to "remove". We definitely shouldn't remove it just yet.